tests/framework/db/QueryTest.php

```diff
@@ -37,6 +37,26 @@ public function testSelect()
         $query->select('id, name');
         $query->addSelect('email');
         $this->assertEquals(['id', 'name', 'email'], $query->select);
+
+        $query = new Query();
+        $query->select('name, lastname');
```

Comment:
Need a test for `$query->select('name, name, name as X, name as X')`

Reply:
Is it real case when programmer duplicates select parts for purpose?

Reply:
Typos do happen.